internal/driver/driver.go

```diff
@@ -335,7 +275,7 @@ func (d *Driver) handleReadCommands(devName string, p protocolMap, reqs []dsMode
 // command.
 func (d *Driver) HandleWriteCommands(devName string, p protocolMap, reqs []dsModels.CommandRequest, params []*dsModels.CommandValue) error {
 	d.lc.Debug(fmt.Sprintf("LLRP-Driver.HandleWriteCommands: "+
-		"device: %s protocols: %v reqs: %+v", devName, p, reqs))
+		"device: %s protocols: %+v reqs: %+v", devName, p, reqs))
```

Comment:
`p` is a `map`, no `+` needed.
```suggestion
		"device: %s protocols: %v reqs: %+v", devName, p, reqs))
```

Reply:
Use `%s` for `devName` and just `%v` for `p`

Reply:
removed + for `p`